src/main/java/wolox/training/models/UserBook.java

```diff
@@ -13,11 +13,14 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.ManyToMany;
+import lombok.Data;
+import lombok.NonNull;
 
 /**
  * This class permits to manage user´s books
  */
 
+@Data
 @Entity
 @ApiModel(description = "Users, who are book's owner")
 public class UserBook {
```

Comment:
Please delete no args constructor and use the annotation @NoArgsConstructor of lombok to avoid boilerplate code

Reply:
It was fixed at last commit